controllers/user.server.controller.js

```diff
@@ -21,14 +31,14 @@ exports.userCreate = function(req, res) {
         forumUserParams;
 
     // Check that every expected forum user attribute exists in the request body
-    if (!reqBody.username || !reqBody.displayName || !reqBody.email || !reqBody.password) {
+    if (!reqBody.username || !reqBody.email || !reqBody.password) {
         isBadRequest = true;
     }
 
     if (!isBadRequest) {
         forumUserParams = {
             'username': reqBody.username.length < 3 ? false : reqBody.username,
-            'displayName': reqBody.displayName.length < 3 ? false : reqBody.displayName,
+            'displayName': generateDisplayName(),
```

Comment:
```suggestion
            'displayName':  reqBody.displayName.length < 3 ? generateDisplayName() : reqBody.displayName,
```
I think the generateDisplayName() function should only be used when a displayName is not provided